vue-custom-bootstrap/src/views/AssessmentSectionList.vue

```diff
@@ -1,33 +1,38 @@
 <template>
-	<nav id="assessment-section-list"> 
-    <b-nav vertical class="w-20">
-
-      <b-nav-item 
-      v-for="item in assessmentData" v-bind:key="item.id"
-      :href="getHref(item.id, item.title)">
```

Comment:
Switched from using a nav-item to a list group because it looked weird on mobile screen sizes. With the v-b-scrollspy directive, vue-bootstrap automatically adds the "active" state while we scroll as well.

Reply:
The "sticky-top" utility class adds position: sticky to get the sidebar sticking to the top but not hardcoded: https://developer.mozilla.org/en-US/docs/Web/CSS/position

Reply:
Looks good. I guess css can be used to make listgroup look like the nav-item, if need be

Reply:
I have no issues with these changes

Reply:
Ok, thanks for taking a look!